output/benchmarking/subset_forecasts_by_location.tsv

```diff
@@ -1,9 +1,15 @@
 crps_hosp	crps_ww	bias_hosp	bias_ww	ae_hosp	ae_ww	location	wweval_commit_hash	wwinference_version	time_stamp
-1.0376521512900907	1.0118669366748054	-0.036991228070175436	-0.007656798245614037	1.4245526802625763	1.4264265157766278	AK	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
-24.626800317489625	19.49484775911998	0.2060657894736842	0.4117872807017544	30.689131156635455	24.783400147179314	MA	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
-2.9839338399865656	1.8872799196790222	-0.01010087719298246	0.04486293859649123	4.248055546108953	2.2765310519617756	NH	51ddee5	227-inf-feedback	2024-10-25T17:16:51Z
+0.8965142016800669	0.8905017309335903	-0.08441447368421053	-0.05893311403508772	1.1813861754140158	1.196834184074869	AK	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
+30.124724302104855	22.978722725100383	0.18085855263157893	0.29270504385964913	37.75519596178603	29.71667330700359	MA	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
+2.4547733753676817	1.6675886683612644	-0.09213486842105263	-0.030434210526315793	3.3591414035433544	2.273867243701906	NH	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
+67.38497962444544	65.01663625797072	0.22840570175438596	0.2876326754385965	82.28019603051415	80.40082146418932	NJ	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
+4.782970351392961	14.647125753083477	0.028859649122807012	0.33272258771929825	6.888723366598054	18.449366320726124	WA	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
+21.1287923709982	21.040115027089886	0.05231491228070175	0.16473859649122807	26.292928587571122	26.407512503939163	all	72fe69a	227-inf-feedback-int-adj	2024-10-27T13:18:34Z
```

Comment:
@seabbs This is hard to read (we could split it out, thats how I had had it initially). But "all" summarizes across locations and forecasts dates and horizon days (in this case for the subset). 

Going to open an issue to make a summary table/plot of what has changed in this folder as well since squinting at numbers isn't terribly fun